gravity-forms/gw-all-fields-template.php

```diff
@@ -166,6 +166,8 @@ public function all_fields_extra_options( $value, $merge_tag, $modifiers, $field
 					return '';
 				};
 				add_filter( 'gform_order_summary', $func );
+				// Hide "Order Summary" label if nopricingfields is used
+				add_filter( 'gform_display_product_summary', '__return_false' );
```

Comment:
I'm curious if this will cause issues if for some reason you use `{all_fields:nopricingfields}` and  `{all_fields}`  in the same confirmation/notification. 

If so, you may want to remove this filter right before `return $value`.

Reply:
@claygriffiths Tested and fixed by removing filter before returning `$value`.